tools/crm_simulate.c

```diff
@@ -421,21 +421,24 @@ setup_input(const char *input, const char *output)
         cib_conn = cib_new();
         rc = cib_conn->cmds->signon(cib_conn, crm_system_name, cib_command);
 
-        if (rc == pcmk_ok) {
+        if (rc == pcmk_rc_ok) {
```

Comment:
Do the legacy2rc conversion immediately after getting the rc, it'll help make things clearer and will be easy/obvious to delete if the function is converted to the new codes